lib/alerts_viewer_web/live/delay_alert_algorithm/median_component.ex

```diff
@@ -1,12 +1,18 @@
-defmodule AlertsViewerWeb.DelayAlertAlgorithms.MedianComponent do
+defmodule AlertsViewer.DelayAlertAlgorithm.MedianComponent do
   @moduledoc """
   Component for controlling the Median delay alert recommendation algorithm.
   """
 
   use AlertsViewerWeb, :live_component
 
+  @behaviour AlertsViewer.DelayAlertAlgorithm
+
   alias Routes.{Route, RouteStats}
 
+  @snapshot_median_min 50
```

Comment:
We can probably use these values on lines 45 and 46 too

Reply:
Yep, that's a good point. Wired up a couple private functions to pipe those through.